test/unit/files/test_ftp.py

```diff
@@ -22,7 +22,7 @@ def test_file_source_ftp_specific():
     assert_realizes_contains(
         file_sources,
         test_url,
-        "This is ftp.gnu.org, the FTP server of the the GNU project.",
+        "This is ftp.gnu.org, the FTP server of the GNU project.",
```

Comment:
Can you please revert this change? The typo comes from the remote ftp.gnu.org/README

Reply:
Sure thing.